pkgs/by-name/po/pollymc-unwrapped/package.nix

```diff
@@ -0,0 +1,118 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  cmark,
+  darwin,
+  extra-cmake-modules,
+  gamemode,
+  ghc_filesystem,
+  jdk17,
+  kdePackages,
+  ninja,
+  nix-update-script,
+  stripJavaArchivesHook,
+  tomlplusplus,
+  zlib,
+
+  msaClientID ? null,
+  gamemodeSupport ? stdenv.isLinux,
+}:
+
+let
+  libnbtplusplus = fetchFromGitHub {
+    owner = "PrismLauncher";
+    repo = "libnbtplusplus";
+    rev = "a5e8fd52b8bf4ab5d5bcc042b2a247867589985f";
```

Comment:
Is there any hint somewhere upstream which commit they are using?

Reply:
nope, but i suggest - last commit, as i've built both prism launcher and pollymc with this option and got last commit from repo.

Reply:
i'll leave it like this, bc after two more test builds with `PolyMC/libnbtplusplus` and `MultiMC/libnbtplusplus` nothing changed, binary builded and worked just fine